src/main/java/io/quarkus/search/app/quarkusio/QuarkusIO.java

```diff
@@ -81,107 +84,243 @@ public static Path yamlQuarkiverseMetadataPath(String version) {
         return Path.of("_data", "versioned", version.replace('.', '-'), "index", "quarkiverse.yaml");
     }
 
-    private final URI webUri;
-    private final GitCloneDirectory mainRepository;
-    private final Map<Language, GitCloneDirectory> localizedSites;
+    private final Map<Language, GitCloneDirectory> allSites;
     private final Map<Language, URI> localizedSiteUris;
     private final CloseableDirectory prefetchedGuides = CloseableDirectory.temp("quarkiverse-guides-");
     private final FailureCollector failureCollector;
 
     public QuarkusIO(QuarkusIOConfig config, GitCloneDirectory mainRepository,
             Map<Language, GitCloneDirectory> localizedSites, FailureCollector failureCollector) throws IOException {
-        this.webUri = config.webUri();
-        this.mainRepository = mainRepository;
-        this.localizedSites = Collections.unmodifiableMap(localizedSites);
-        this.localizedSiteUris = localizedSites.entrySet().stream().collect(
-                Collectors.toUnmodifiableMap(Map.Entry::getKey, e -> config.localized().get(e.getKey().code).webUri()));
+        HashMap<Language, URI> languageUriMap = new HashMap<>(localizedSites.entrySet().stream().collect(
+                Collectors.toMap(Map.Entry::getKey, e -> config.localized().get(e.getKey().code).webUri())));
+        languageUriMap.put(Language.ENGLISH, config.webUri());
+        this.localizedSiteUris = Collections.unmodifiableMap(languageUriMap);
```

Comment:
Feels like this should now be renamed to `languageUriMap` or `siteUris`? Since it's not about "localized sites" anymore.